python/README.md

```diff
@@ -1,64 +1,22 @@
-# Installation
+# AIF Python API
 
-To use the `aida_interchange` package within your project you must first install it using the instructions described in the [Install setup.py](install-setup.py) section. It is recommended that you install `aida_interchange` into a virtual environment for development purposes. To install `aida_interchange`  within your virtual environment first activate your virtual environment and run the install command below. See [Python Development Setup](python-development-setup) for more details on creating a virtual environment.
+To use the `aida_interchange` package within your project, you must first install it. It is recommended that you install `aida_interchange` into a python3 virtual environment. See [Python Virtual Environment README](VIRTUAL-README.md) for more details on creating and using a virtual environment.
 
-# Install setup.py
+## Install setup.py
 
-To install `aida_interchange`, run the following command within the `python` directory:
+To install `aida_interchange`, make sure a [Python Virtual Environment](VIRTUAL-README.md) is activated, navigate to the `AIDA-Interchange-Format/python` directory, and run the following command:
```

Comment:
Is the `AIDA-Interchange-Format` context necessary here?  Also, there's no requirement that the directory is named like that, even if it's the default when you clone the repo.

Reply:
Good point. It isn't as necessary here, but in the Verdi repo, I thought it was important because we cross-reference to the AIF repo, and we have multiple python directories.  I was trying to mirror that here, but maybe it isn't (as) necessary.

Reply:
Maybe not as necessary, but it doesn't hurt either.